etl/steps/data/garden/artificial_intelligence/2023-06-14/ai_robots.py

```diff
@@ -110,6 +99,9 @@ def run(dest_dir: str) -> None:
     # Merge pivot table for professional service robots, application area and sector with aggregates
     merge_all = pd.merge(merge_service, df_agg_clean, on=["year", "country"], how="outer")
 
+    # Rename Chinese Taipei to Taiwan
+    merge_all.loc[merge_all["country"] == "Chinese Taipei", "country"] = "Taiwan"
```

Comment:
Would it be better to use the country harmonizer tool? Maybe you have a good reason for not using it in this instance.